lib/graphql/schema/wrapper.rb

```diff
@@ -0,0 +1,26 @@
+module GraphQL
+  class Schema
+    class Wrapper
+      include GraphQL::Schema::Member::CachedGraphQLDefinition
+      include GraphQL::Schema::Member::TypeSystemHelpers
+
+      attr_reader :of_type
+      
+      def initialize(of_type)
+        @of_type = of_type
+      end
+
+      def to_graphql
+        raise NotImplementedError
+      end
+
+      def unwrap
+        @of_type.unwrap
+      end
+
+      def to_type_signature
+        "#{@of_type.to_type_signature}!"
```

Comment:
Oops, I think this one should stay in the child classes, since `List`'s implementation is different.

Reply:
Oops, totally! Nice catch, thank!